include/onnxruntime/core/platform/windows/TraceLoggingConfigPrivate.h

```diff
@@ -0,0 +1,2 @@
+#define TraceLoggingOptionMicrosoftTelemetry() \
```

Comment:
This file will be deleted.

Reply:
Still needs to be deleted

---
In reply to: [355012639](https://github.com/microsoft/onnxruntime/pull/2577#discussion_r355012639) [](ancestors = 355012639)